core/src/test/java/it/pagopa/pdv/person/core/PersonServiceImplTest.java

```diff
@@ -27,14 +27,13 @@ class PersonServiceImplTest {
     @Mock
     private PersonConnector personConnector;
 
-
     @Test
     void findById_nullId() {
```

Comment:
add a test case `findById_nullNamespace` like this one

Reply:
added with https://github.com/pagopa/pdv-ms-person/pull/42/commits/4c55bfe44b3119678e0444a039f44b894afafdc1